docs/renater_federation_saml.md

```diff
@@ -0,0 +1,71 @@
+### Renater federation use
```

Comment:
I think we need to specify "Renater's FER Shibboleth federation" because they have several?

Reply:
Yes :)